charts/rucio-ui/values.yaml

```diff
@@ -11,13 +11,25 @@ useDeprecatedImplicitSecrets: true
 exposeErrorLogs: true
 
 service:
-  type: NodePort
   # Run the webui server on port 443 instead of 80 and accept X509 certificates and proxies
```

Comment:
```suggestion
```
This comment is out of date now

Reply:
Not sure what changed. 443 is still required if you need to use x509 auth :)